### ISSUE TITLE
Ragged Tensors

### ISSUE THREAD

**Issue by PatReis** [keras-team/keras-core#803]
Hello,

thanks for the wonderful work on Keras-Core.
I saw the issue #670 and #223 and wanted to ask about the ideas/roadmap to support ragged tensors.
Will there be a possibilty to pass a `KerasRaggedTensor` to layers and models without the support of ragged operations across backends of course? To become compatible with `tf.keras` ?
I know that this is probably a lot of work, so I just wanted to know to plan ahead.

Best regards

**Comment by fchollet** [comment]
Report from the other thread:

We *might* support `RaggedTensor` in the future with the TF backend specifically (such a feature does not exist with other frameworks).

However if you're just looking to have a dynamic data dimension, then you can do it just by:

- Bucketing your samples into buckets of shape `(batch, A, ...)`, `(batch, B, ...)`, etc. and creating batches out of the buckets, so that each batch is rectangular
- Padding/truncating your samples to a shared shape
- Some combination of the two

In general it is possible to handle any workflow using rectangular tensors. RaggedTensors are a convenience but not a blocker.
